### ISSUE TITLE
Case-insensitive organization names

### ISSUE THREAD

**Issue by dylanpyle** [Shyp/go-circle#1]
Since GitHub and Circle treat organization names as case-insensitive, it would be great if this tool did the same.

I'm not very consistent about my naming when I'm originally setting up origins, so I end up with different repos configured different ways, e.g.

```
git@github.com:Shyp/go-circle.git
git@github.com:shyp/something-else.git
```

I could fix these, but it's not technically incorrect as far as I'm aware (plus, _so much effort_).

Unfortunately that means I have to have two entries in my circle config:

```
[organizations]
    [organizations.shyp]
    token = "<snip>"

    [organizations.Shyp]
    token = "<snip>"
```

Otherwise:

```
λ circle wait
Waiting for latest build on my-cool-branch to complete
Couldn't find organization Shyp in the config
```

**Comment by kevinburkeshyp** [comment]
Ahh, interesting. Will have to think about how to implement this!

**Comment by kevinburkeshyp** [comment]
@dylanpyle just pushed a fix here! update to Circle 0.11 or Shyp 0.6
